index.html

```diff
@@ -9,27 +9,72 @@
 </head>
 <body>
     <div class="parent-div">
-        <section class="main-page">
-            <h1 class="main-heading">All Awesome Books</h1>
-            <div class="all-books-container">
-                <!-- Code written in JS -->
+        <header>
+            <nav>
+                <h2 class="logo">Awesome Books</h2>
+                <ul class="nav-links-container">
+                    <li class="nav-links nav-list">List</li>
+                    <li class="nav-links nav-add-new">Add New</li>
+                    <li class="nav-links nav-contact">Contact</li>
```

Comment:
- [ ] [OPTIONAL] Great job implementing the navigation :+1: I think it would be more convenient if the cursor changes to a pointer when a user hovers over the navigation links. It'll result in a better user experience by letting users know that the links are clickable. 🤝️